components/sections/Navbar/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as AccessibleIcon from '@radix-ui/react-accessible-icon';
+import Image from 'next/image';
+import { useTheme } from 'next-themes';
+import { useMemo } from 'react';
+
+import LanguageDropdown from '@/components/Common/LanguageDropDown';
+import ThemeToggle from '@/components/Common/ThemeToggle';
+import NavItem from '@/components/sections/NavItem';
+import { useLocale } from '@/hooks/useLocale';
+import { useNavigation } from '@/hooks/useNavigation';
+
+const Navbar: React.FC = () => {
+  const { newNavigationItems } = useNavigation();
+  const { availableLocales, currentLocale } = useLocale();
+  const { theme, setTheme } = useTheme();
+
+  const GHLogoSrc = `/static/images/logos/social-github${
+    theme === 'light' ? '-dark' : ''
+  }.svg`;
+
+  const NodeLogoSrc = `/static/images/logos/horizontal${
+    theme === 'light' ? '-dark' : '-light'
+  }.svg`;
+
+  const availableLanguages = useMemo(
+    () =>
+      availableLocales.map(locale => ({
+        name: locale.name,
+        code: locale.code,
+      })),
+    [availableLocales]
+  );
+
+  const currentLanguage = useMemo(
+    () => ({
+      name: currentLocale.name,
+      code: currentLocale.code,
+    }),
+    [currentLocale]
+  );
+
+  return (
+    <nav className="flex h-16 justify-between border-b-2 border-neutral-200 px-8 dark:border-0">
```

Comment:
Read [How to style a component](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#styling-a-component) on our contribution guideline

Reply:
hey, I missed a part here. I meant to draft this PR but still wanted reviews as I had some concerns. Since adding a class on the markup itself is convenient during development, I decided to create css modules after all styles are fixed.

Reply:
Hey, I decided to mark this PR ready for review, I'll create modules css right away.